test/HiIQRewards.test.ts

```diff
@@ -9,13 +9,20 @@ import {
 import {setupUser, setupUsers} from './utils';
 
 const setup = deployments.createFixture(async () => {
-  await deployments.fixture();
+  await deployments.fixture('HIIQ');
   const {deployer} = await getNamedAccounts();
+  const HIIQ = <HIIQ>await ethers.getContract('HIIQ');
+  const IQERC20 = <IQERC20>await ethers.getContract('IQERC20');
+  await deployments.deploy('HiIQRewards', {
+    from: deployer,
+    args: [IQERC20.address, HIIQ.address],
```

Comment:
@JustinCast the problem was that HIIQ in the deployment file its a namedAccount (because of the issue w vyper contracts and hardhat deployment).

In this case, on executing the fixtures, it was creating a HiIQRewards with a HiIQ address in the constructor that wasn't the one with the real code. The best option for this cases its just to add the deploy to the test setup()

One way of debugging it, it was using console.logs at contract level. You can do that following this tutorial https://hardhat.org/tutorial/debugging-with-hardhat-network.html#solidity-console-log

I didn't check the commented tests, but feel free to continue uncommenting them and making sure they pass.